.pre-commit-hooks.yaml

```diff
@@ -32,6 +32,12 @@
     entry: check-docstring-first
     language: python
     types: [python]
+-   id: check-encoding
+    name: check text files have desired encoding
+    description: checks that text files decode cleanly using an encoding.
+    entry: check-encoding
+    language: python
```

Comment:
Why only python? 🤔

Reply:
`language` means what the hook is written in. One line below is what the hook applies _to_:
```
types: [text]
```

Reply:
🤦🏼‍♂️ sorry